src/test/javascript/general.controller.test.js

```diff
@@ -209,12 +209,12 @@ describe("GeneralController", function () {
         });
         it("should set selectedGrouping to 'zzzz'", () => {
             scope.selectedGrouping = 0;
-           scope.displayGrouping(0, 0);
-           expect(scope.selectedGrouping).toEqual('zzzz');
+            scope.displayGrouping(0, 0);
```

Comment:
Why was a space added in front of these lines?

Reply:
weird, I didnt make these changes explicitly, don't know what went wrong here but I'll fix it.